service/src/main/java/bio/terra/cbas/dependencies/github/GitHubService.java

```diff
@@ -1,18 +1,28 @@
 package bio.terra.cbas.dependencies.github;
 
+import bio.terra.cbas.dependencies.ecm.EcmService;
 import org.springframework.stereotype.Component;
 
 @Component
 public class GitHubService {
   private final GitHubClient client;
+  private final EcmService ecmService;
 
-  public GitHubService(GitHubClient gitHubClient) {
+  public GitHubService(GitHubClient gitHubClient, EcmService ecmService) {
```

Comment:
my first reaction to this was: is it weird to have a service contain another service?

this might be premature optimization, but I wonder if it makes sense to have a service that is responsible for external workflow repositories. For now that's just github, but maybe having all the logic in the same place will make sense when we have more locations to choose from.

I don't think it's a problem as written, though.